app/models/mobility_string_translation.rb

```diff
@@ -0,0 +1,16 @@
+class MobilityStringTranslation < ApplicationRecord
+  
+  belongs_to :translatable, polymorphic: true
+  belongs_to :created_by, class_name: "User", optional: true
+
+  validates :locale, presence: true
+  validates :key, presence: true
+  validates :value, presence: true
+
+  validates :translatable_id, uniqueness: { scope: [:translatable_type, :locale, :key] }
```

Comment:
This enforces the index constraint set by mobility gem install on the Rails side